packages/ingest/src/steps/0-start.ts

```diff
@@ -47,12 +51,12 @@ export async function ingestStart({
       await importLessonsFromCSV({
         ingestId,
         filePath: config.source.filePath,
-        onError: console.error,
+        onError: (error) => log.error(error.message),
```

Comment:
Have you looked at providing the message object itself rather than the error? It should work